extensions/core/deployment/src/main/java/io/quarkiverse/cxf/deployment/CxfBuildTimeConfig.java

```diff
@@ -445,4 +445,116 @@ default void validate(String prefix) {
         }
     }
 
+    /**
```

Comment:
Could we please move this to a new class in the security extension?

I'd vote for it being a Runtime fixed setting (should go to Security runtime module), so that users can query the values at runtime from their apps. 

Something like 

```
@ConfigMapping(prefix = "quarkus.cxf")
@ConfigRoot(phase = ConfigPhase.BUILD_AND_RUN_TIME_FIXED)
@ConfigDocFilename("quarkus-cxf-rt-ws-security.adoc")
public interface CxfSecurityFixedConfig {
...
```